apps/src/experiments.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * This module contains logic for tracking various experiments. Experiments
+ * can be enabled/disabled one of two ways
+ * (1) enable: localStorage.set('experiments-experimentName', true)
+ *     disable: localStorage.removeItem('experiments-experimentName')
+ * (2) Add a query param, i.e. http://foo.com?experimentName=true or
+ *     http://foo.com?experimentName=false
+ * The latter approach ends up toggling the localStorage state.
+ */
+var experiments = module.exports;
+
+/**
+ * Checks whether provided experiment is enabled or not
+ * @param {string} key - Name of experiment in question
+ * @returns {bool}
+ */
+experiments.isEnabled = function (key) {
+  var enabled;
+  var queryString = this.__TestInterface__.queryString ||
```

Comment:
At minimum, extracting a `getQueryString` helper could hide the kludge.

Then, you might be able to have the test monkeypatch over the `getQueryString` method, so you don't need to expose any special test interface here.  I'm not sure if that's 'cleaner' than what you already have though.

It's too bad you can't just overwrite `window.location.search` since you already have to run this test in a browser context.  You could always set up this method so that you have to inject `window` into it, but that's a step away from the simplicity of your interface.